### PR TITLE
[JENKINS-13545] use dialog to delete a folder

### PR DESCRIPTION
**Pull request by mawinter69**
See [JENKINS-13545](https://issues.jenkins-ci.org/browse/JENKINS-13545).

### Proposed changelog entries

* Entry 1: use dialog to delete a folder
* ...

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change).
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests

### CODE REVIEW

**Review by jglick — APPROVED** [review]

File: src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties [hunk 1]
```diff
@@ -20,4 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-delete={0}\u306e\u524a\u9664
+delete={0}\u306E\u524A\u9664
+delete.confirm={0} "{1}"\u3092\u524A\u9664\u3057\u3066\u3082\u3088\u308D\u3057\u3044\u3067\u3059\u304B?
```
Comment (mawinter69) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties:24]: I hope that the translation is correct. I copied this from the configurable.jelly from core

Comment (jglick) [reply src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties:24]: As a rule, if you are not a native speaker, you should not add or edit entries in translations. If a given key is deleted from the base (English) file, or changes meaning or format, delete it from translations. Let translators keep up later if they want.

(https://github.com/jenkinsci/jep/blob/master/jep/216/README.adoc would be clearer I think.)

File: src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_de.properties [hunk 2]
```diff
@@ -0,0 +1,13 @@
+AbstractFolder.BeingDeleted={0} wird gerade gel\u00F6scht
```
Comment (jglick) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_de.properties:1]: I would have to look up the specific parent / core versions, but using recent deps/tools I think you can switch to UTF-8. Check https://github.com/jenkinsci/jenkins/pull/6719 https://github.com/jenkinsci/jenkins-test-harness/pull/448

File: src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties (hunk 1 above)
Comment (mawinter69) [reply src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties:24]: ok removed it

File: src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties [hunk 3]
```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-delete={0}\u306e\u524a\u9664
+delete={0}\u306E\u524A\u9664
```
Comment (jglick) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties:23]:
```suggestion
delete={0}\u306e\u524a\u9664
```

just to leave it untouched

**Review by olamy — APPROVED** [review]

**Review by rsandell — APPROVED** [review]

**Review by car-roll — APPROVED** [review]

**Comment by mawinter69** [comment]
Caused ath failures. Fixed with https://github.com/jenkinsci/acceptance-test-harness/pull/1394

**Pull request merged**